csaf_2.1/prose/edit/src/distributing.md

```diff
@@ -597,6 +597,17 @@ The temporary blocking SHOULD be removed as soon as possible, at latest two week
 
 > Also confer to the TC's guidance on content delivery networks and caching.
 
+### Requirement 25: Access-Control-Allow-Origin
+
+For any CSAF documents and related metadata, the web server SHOULD set the HTTP header `Access-Control-Allow-Origin: *`.
+
+> The HTTP header enables users to access the CSAF data with web browser based clients.
+
+The HTTP header MAY be altered to allow just specified domains.
+In such case, the response SHOULD also include the `Vary` header.
```

Comment:
The recommendation for the `Vary` header here is more complex. 
https://fetch.spec.whatwg.org/#cors-protocol-and-http-caches explains that setting `Vary: Origin` is only good, if there are two or more domains. 

So we could either:
 1. refer to that recommendation, like ``In such case follow the response SHOULD follow the recommendations of [FETCH 6] about setting the `Vary` header for caching.``

BTW: I wonder if caching really is relevant here as protected feeds will have to send credentials via TLS anyway. But in theory caching could happen before the TLS termination on the servers side.

2. Be more specific: ``In case of two or more domain, the response SHOULD also include a `Vary: Origin` header.``

Reply:
Addressed with [fa234ce](https://github.com/oasis-tcs/csaf/pull/903/commits/fa234cebca8b99a0d6a22916937895774c48a44f)

Reply:
LGTM